import-export-cli/integration/base/helper.go

```diff
@@ -379,6 +379,7 @@ func GetExportedPathFromOutput(output string) string {
 	//Check directory path to omit changes due to OS differences
 	if strings.Contains(output, ":\\") {
 		arrayOutput := []rune(output)
+		fmt.Println("Array Output: ", arrayOutput[0])
```

Comment:
Shall we remove this?

Reply:
Fixed by [59b2698](https://github.com/wso2/product-apim-tooling/pull/925/commits/59b2698c4e9aceab8b212ba2afd761955611f7e4)